logs.py

```diff
@@ -1,9 +1,37 @@
+from pubnub import Pubnub
+
 from .base_request import BaseRequest
-from .resources import Message
+from .models.config import Config
+
+class Logs(object):
+	def __init__(self):
+		self.config = Config()
+
+	def init_pubnub(func):
+		def wrapper(self, uuid, callback, error):
```

Comment:
This method shouldn't care about the wrapped function arguments, so can be `wrapper(*args, **kwargs)`?